travel_platform/izitravel_tests.py

```diff
@@ -0,0 +1,79 @@
+# coding=utf-8
+import requests
+import unittest
+
+from conf import config
+from izi_travel_client import IziTravelApiClient, IziTravelApiError
+
+
+class IziTravelTest(unittest.TestCase):
+    def setUp(self):
+        self.client = IziTravelApiClient(api_key=config.get(
```

Comment:
Just FYI:
`setUp` method runs before each class. It mean it connects to API n times, where n equals number of tests.

There is also one amazing feature called `setUpClass`. Sometimes it may be really needed for some reason.
More info may be found here:
https://docs.python.org/2/library/unittest.html#class-and-module-fixtures
https://stackoverflow.com/a/14044585/5490451